ShellBags/v2
# website front page

there is no functionality in any of the buttons, but the visuals for the front page are finished based on the wireframe in the design document

Steps to test:
- npm install within the websiteV2 folder
- npm start
- see changes in http://localhost:3000/

Known bugs:
currently when i run the website, there's a small margin around all the page content (visible around the top bar). i thought i had fixed that but it reverted after my last PR i guess, either i need to fix it or i may potentially leave it? feedback would be welcomed for that

![image](https://user-images.githubusercontent.com/54541520/105921954-a6898500-6007-11eb-96d3-c35706d2edea.png)
